addons/event/models/event_ticket.py

```diff
@@ -55,6 +57,7 @@ def default_get(self, fields):
             res['name'] = _('Registration for %s', self.env.context['default_event_name'])
         return res
 
+    sequence = fields.Integer('Sequence', default=20)
```

Comment:
Why `20` instead of the inherited `10` ?

Reply:
I think we shouldn't care to have the best codebase quality for production use while supporting demo data.
As `10` is the usually used default, let's try and make it work. 
If we don't have ten tickets per event, could we give the same sequence in demo data for tickets across different events (as they are never shown together and `_order` uses additional fields)? To check if nothing breaks when resequencing.
Alternatively, we could give really high values for demo tickets.

Reply:
I would say we set 1,2,... on each event since they do not appear together. That makes sense with the sequence = 10, and seems the shorter / cleanest.

Reply:
LGTM. (If I were picky I would have argued for `5, 6,...` :laughing: )